Android/app/src/main/java/com/example/androidgui/inventoryactivities/AddInventoryActivity.java

```diff
@@ -36,14 +41,22 @@ public void onClick(View v) {
         String iamount = amount.getText().toString();
         String idate = date.getText().toString();
         String ifreshness = freshness.getText().toString();
+        String[] paras = this.updateInventoryList(iname, iprice, iamount, idate, ifreshness);
+        im.addNewInventory(paras);
+    }
+    @Override
+    public String[] updateInventoryList(String name, String price, String amount, String date, String freshness){
```

Comment:
If this is a helper method, this should be private. Public methods in activity shouldn't be returning anything